app/lib/submission_builder/state1099_r.rb

```diff
@@ -0,0 +1,53 @@
+module SubmissionBuilder
+  class State1099R < SubmissionBuilder::Document
+    include SubmissionBuilder::FormattingMethods
+
+    def document
+      form1099r = @kwargs[:form1099r]
+      state_abbreviation = form1099r.intake.state_code.upcase
+
+      build_xml_doc("IRS1099R", documentId: "IRS1099R-#{form1099r.id}") do |xml|
+        xml.PayerNameControlTxt form1099r.payer_name_control
+        if form1099r.payer_name && form1099r.payer_name != ''
+          xml.PayerName do
+            xml.BusinessNameLine1Txt sanitize_for_xml(form1099r.payer_name.tr('-', ' '), 75)
+          end
+          xml.PayerUSAddress do
+            xml.AddressLine1Txt sanitize_for_xml(form1099r.payer_address_line1, 35) if form1099r.payer_address_line1.present?
+            xml.AddressLine2Txt sanitize_for_xml(form1099r.payer_address_line2, 35) if form1099r.payer_address_line2.present?
```

Comment:
got a bunch of errors saying that `""` is not an acceptable type for these fields, which is why I'm adding a `.present?` check for some of these fields.